test/unit-test.el

```diff
@@ -229,3 +230,78 @@ then run BODY."
    "def bar /* foo */"
    (search-forward "foo")
    (should (not (memq 'font-lock-string-face (faces-at-point))))))
+
+(ert-deftest groovy-highlight-variable-assignment ()
```

Comment:
Do you think it would be worthwhile to set a defcustom option for this? Some people (me) might like it.